wallet/src/main/scala/org/bitcoins/wallet/models/AddressTagTable.scala

```diff
@@ -0,0 +1,47 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.protocol.BitcoinAddress
+import org.bitcoins.core.wallet.utxo.AddressTag
+import slick.jdbc.SQLiteProfile.api._
+import slick.lifted.{PrimaryKey, ProvenShape}
+
+abstract class AddressTagDb[AddressTagType <: AddressTag] {
+  def address: BitcoinAddress
+
+  def tag: AddressTagType
```

Comment:
Isn't this idea to have a `Vector[AddressTagType]`?

Reply:
This is for one db row, so there is only one. There is a Vector of AddressTagDAOs